README.md

```diff
@@ -23,11 +23,48 @@ Note: As of 2023-12-14 all but one server do not allow web applications
  due to _CORS restrictions_ often.
  See https://github.com/oasis-tcs/csaf/issues/653 for more details.
 
-The envisoned usage is to be integrated in a larger application.
-Therefore it is kept simple and stylable.
 A backend can act as a proxy to avoid the problems caused by
 CSAF Providers missing `Access-Control-Allow-Origin: *` headers.
 
+The envisoned usage is to be integrated in a larger application.
+Therefore it is kept simple and stylable.
+
+## Feedback sought
+
+> [!NOTE]
+> If you have tried or considered using this component
+> please contact us and explain your use cases.
+>
+> Please also let us know in case you are using webview and the reasons for its selection.
+>
+> Open an issue in this repo or send us an email, e.g. to
+> [@bernhardreiter](https://github.com/bernhardreiter).
+
+Background: Initially this web component was used
+in [ISDuBA](https://github.com/ISDuBA/ISDuBA), which should serve
+as our example for a universal CSAF document display component.
+During ISDuBA's development we decided to integrate a copy
+of Webview's code and make it ISDuBA specific instead.
+We wanted to understand more about
+what a real application handling CSAF documents would want
+from a viewing component.
+
+Our lesson learned: It is hard to display CSAF documents well;
```

Comment:
Please rephrase stressing the challenge of use-case independent, generic viewers.